### ISSUE TITLE
'chai' should be listed in the project's dependencies, not devDependencies

### ISSUE THREAD

**Issue by renanol** [AtomLinter/linter-eslint#790]
## Issue Type

Bug

## Issue Description

When do `import { expect } from 'chai'` linter eslint show the error below, but when run eslint don't show this error. 

Check [link](https://github.com/benmosher/eslint-plugin-import/issues/602) I believe that was the reason for this error.

```
Linter Eslint: Debug output here
```
ESLint
Error
import/no-extraneous-dependencies 'chai' should be listed in the project's dependencies, not devDependencies.

**Comment by Arcanemagus** [comment]
Can you post the debug output as requested? Without that or a example repo/gist it's hard to reproduce this to diagnose it.

**Comment by IanVS** [comment]
This seems to be a duplicate of https://github.com/AtomLinter/linter-eslint/issues/771.  Please take a look there for a description of why this happens and a possible workaround.
